lib/github.com/mpllang/mpllib/MeshToImage.sml

```diff
@@ -50,12 +50,7 @@ struct
       val width = resolution
       val height = resolution
 
-      val niceGray = Color.hsva {h=0.0, s=0.0, v=0.88, a=1.0}
-      (* val white = Color.hsva {h=0.0, s=0.0, v=1.0, a=1.0} *)
       val black = Color.hsva {h=0.0, s=0.0, v=0.0, a=1.0}
-      val red = Color.hsva {h=0.0, s=1.0, v=1.0, a=1.0}
-
-      val niceRed = Color.hsva {h = 0.0, s = 0.55, v = 0.95, a = 0.55}
```

Comment:
These should be left in. These are alternatives that maybe we want to use instead.